### DISCUSSION TITLE
Trouble listing scanned Wifi Networks

### DISCUSSION THREAD

**Discussion by adelbke** [3xploitGuy/lazyaircrack#8] [Q&A]
I made sure to run the script with root privileges
As you can see in the wifi list there's a cut and wc error and a  generated-01.kismet.csv that is missing

```
[Selected] Option 1 Wifi Hacking...
ioctl(SIOCGIFINDEX) failed: No such device
[wlan0mon] Preparing for scan...
./lazyaircrack.sh: line 141: kill: (13480) - No such process
sed: can't read generated-01.kismet.csv: No such file or directory


SerialNo        WiFi Network
cut: generated-01.kismet.csv: No such file or directory
wc: generated-01.kismet.csv: No such file or directory
./lazyaircrack.sh: line 147: [: 1000: unary operator expected
./lazyaircrack.sh: line 151: generated-01.kismet.csv: No such file or directory
./lazyaircrack.sh: line 152: generated-01.kismet.csv: No such file or directory
./lazyaircrack.sh: line 153: generated-01.kismet.csv: No such file or directory

[] Preparing for attack...
[wlan0mon] Sending DeAuth to target...
QStandardPaths: XDG_RUNTIME_DIR not set, defaulting to '/tmp/runtime-root'
QStandardPaths: XDG_RUNTIME_DIR not set, defaulting to '/tmp/runtime-root'
QStandardPaths: XDG_RUNTIME_DIR not set, defaulting to '/tmp/runtime-root'

```

**Comment by 3xploitGuy** [comment]
Seems like you are using VirtualMachine, which doesn't allows the OS to use Host WiFi adapter.

Either you can dual boot OS or use an External WiFi adapter (supporting monitor mode) in case you want to continue using VirtualMachine.

Though the tool doesn't support External WiFi adapter yet but there is upcoming update in next 2-3 days which will support this and few more improvements.

**Reply by adelbke** [reply]
This is not a VM, it's my machine a thinkpad t480s that runs the latest Kubuntu release

**Reply by adelbke** [reply]
btw I also found out later that those files (the missing kismet csv's) were in my Home directory instead of the directory containing the script.

**Reply by 3xploitGuy** [reply]
Gotcha, the problem is with the name of WiFi adapter, KaliLinux calls it by wlan0 (currently used in tool) and different OS has different naming.
I'm removing this hard-coded value and reading it by some other way, some final testing and in next version this problem will be solved, mostly tomorrow is the release.

**Reply by 3xploitGuy** [reply]
@Quickinline This problem should be resolved now, you can try cloning new version of script, if you face any issues definitely we can discuss.

**Reply by adelbke** [reply]
Thanks @3xploitGuy I'll check it out

**Comment by mdsabbirkhan5212-crypto** [comment]
<img width="1170" height="765" alt="103147425-ea743a80-477a-11eb-999c-f590f5165ffc" src="https://github.com/user-attachments/assets/253b1f62-1d02-4dab-bc34-012c5d597e3c" />
Bro no root my fon
